src/main/java/ladder/domain/Users.java

```diff
@@ -0,0 +1,51 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Users {
+
+    private final List<User> users;
+
+    private Users(List<User> users) {
+        this.users = users;
+    }
+
+    public static Users create(String users) {
+        return new Users(
+                Arrays.stream(users.split(","))
+                        .map(User::createWithName)
+                        .collect(Collectors.toList())
+        );
```

Comment:
```suggestion
        return Arrays.stream(users.split(","))
                        .map(User::createWithName)
                        .collect(Collectors.collectingAndThen(Collectors.toList(), Users::new));
```

소소하지만 다음과 같이 작성해볼 수도 있습니다 :)

Reply:
꿀팁 감사합니다 :)
제가 작성한 코드보다 더 함수형 스럽게 사용할 수 있네요. 하나 배워갑니다 👍